docs/openapi.yaml

```diff
@@ -48,21 +77,38 @@ paths:
               example:
                 $ref: ./api/faucet/run-faucet.example.json
         500:
-          description: Failed to run faucet
+          description: Faucet call failed
```

Comment:
nit: could say that one potential reason might be that the faucet ran out of tokens?

Reply:
Again, this field is meant for a short string. I can add this information in the endpoint description.